federationapi/routing/send.go

```diff
@@ -116,6 +116,7 @@ func (t *txnReq) processTransaction() (*gomatrixserverlib.RespSend, error) {
 			results[e.EventID()] = gomatrixserverlib.PDUResult{
 				Error: err.Error(),
 			}
+			util.GetLogger(t.context).Error("Failed to process incoming federation event, skipping. Event ID:", e.EventID(), " Err:", err)
```

Comment:
I think this should be something like:
```
util.GetLogger(t.context).WithError(err).Error("Failed to process...")
```

Reply:
Done, and dropped it to warning since it's not a huge problem but may signify a problem, as in this case!